src/jit/lowerxarch.cpp

```diff
@@ -239,16 +239,17 @@ void Lowering::TreeNodeInfoInit(GenTree* tree)
 #if !defined(_TARGET_64BIT_)
 
         case GT_LONG:
-            if (tree->gtNext == nullptr)
+            if ((tree->gtLIRFlags & LIR::Flags::IsUnusedValue) != 0)
```

Comment:
Something happened to observe and feel free to address it as separate PR:
It might be a good idea to abstract this in get/set methods.  source search indicates there are 7 other places where this flag is either set/reset/checked.